lib/cdo/aws/s3.rb

```diff
@@ -19,15 +19,13 @@ def call(context)
           start_time = Time.now
           response = @handler.call(context)
           duration = Time.now - start_time
-          if duration > context.config.notify_timeout
-            Honeybadger.notify(
-              error_class: "SlowAWSResponse",
-              error_message: "Slow AWS response",
-              context: response.context.
-                http_response.headers.to_h.
-                slice('x-amz-request-id', 'x-amz-id-2').
-                merge(duration: duration)
-            )
+          # Only track long S3 API calls issued by web application server code on our production or managed test systems.
+          if CDO.running_web_application? &&
+              (CDO.rack_env?(:production) || CDO.test_system?) &&
+              duration > context.config.notify_timeout
+
+            Cdo::Metrics.put('S3Client', 'SlowResponse', 1, {Host: CDO.dashboard_hostname})
+            Cdo::Metrics.put('S3Client', 'SlowResponseDuration', duration, {Host: CDO.dashboard_hostname})
```

Comment:
Any chance we'll want to compare these against non-slow response duration values (and not have them available because we only log duration when its slow)?